### PR TITLE
passwordresetdoneview

### PR DESCRIPTION
**Pull request by seyiadel**
Added PriceJarPassWordResetDoneView using Class Based Views.

### CODE REVIEW

File: pricejar/base/views.py [hunk 1]
```diff
@@ -52,3 +52,7 @@ class PriceJar_Password_Change_Done_View(PasswordChangeDoneView):
     template_name = "password_change_done.html"
     title = "Password Change Done Successfully"
 #end of password_change_done_view
+
+#password__reset_done_view
+class PriceJarPassWordResetDoneView(PasswordResetDoneView):
+    template_name = "password_reset_sent.html"
```
Comment (mirera) [inline pricejar/base/views.py:58]: Add a closing comment please. To guide other developers

**Review by mirera — APPROVED** [review]
Check out the review

**Comment by mirera** [comment]
Always link your commit, branch with your issue to make it easier for automating the project board.

**Comment by seyiadel** [comment]
> Always link your commit, branch with your issue to make it easier for automating the project board.

Yes.

**Pull request merged**